css/custom.css

```diff
@@ -964,3 +964,10 @@ h1 .tm, h2 .tm, h3 .tm {
     color: #555;
     font-weight: 500;
 }
+
+/* GitHub style blockquote */
```

Comment:
cc @gengliangwang FYI

Reply:
![image](https://user-images.githubusercontent.com/1097932/168199619-add2e183-63e9-4c69-8572-6d17bdc0bf56.png)

Let's try copying the "display: block; marigin-block-start..."

Reply:
🙆‍♂️